packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneBlobs.spec.ts

```diff
@@ -0,0 +1,192 @@
+
+    const loadSummarizerAndContainer = async (summaryVersion?: string) => {
+        return createSummarizerWithContainer(
+            provider,
+            documentAbsoluteUrl,
+            summaryVersion,
+            gcOptions,
+            mockConfigProvider(settings),
+        );
+    };
+    const summarize = async (summarizer: ISummarizer) => {
+        await provider.ensureSynchronized();
+        return summarizeNow(summarizer);
+    };
+
+    beforeEach(async function() {
+        provider = getTestObjectProvider({ syncSummarizer: true });
+        if (provider.driver.type !== "local") {
+            this.skip();
+        }
+        settings["Fluid.GarbageCollection.ThrowOnTombstoneUsage"] = true;
+        settings["Fluid.GarbageCollection.TestOverride.SweepTimeoutMs"] = sweepTimeoutMs;
+    });
+
+    // This function creates an unreferenced blob and returns the blob's id and the summary version that
+    // blob was unreferenced in.
+    const summarizationWithUnreferencedBlobAfterTime =
+    async (approximateUnreferenceTimestampMs: number) => {
+        const container = await makeContainer();
+        const defaultDataObject = await requestFluidObject<ITestDataObject>(container, "default");
+        await waitForContainerConnection(container);
+
+        // Create blob
+        const handleKey = "handle";
+        const blobContents = "Blob contents";
+        const blobHandle = await defaultDataObject._runtime.uploadBlob(stringToBuffer(blobContents, "utf-8"));
+
+        // Reference a blob - important for making it live
+        defaultDataObject._root.set(handleKey, blobHandle);
+
+        // Unreference a blob
+        defaultDataObject._root.delete(handleKey);
+
+        // Summarize
+        const {
+            container: summarizingContainer1,
+            summarizer: summarizer1,
+        } = await loadSummarizerAndContainer();
+        const summaryVersion = (await summarize(summarizer1)).summaryVersion;
+
+        // Close the containers as these containers would be closed by session expiry before sweep ready ever occurs
+        container.close();
+        summarizingContainer1.close();
+
+        // Wait some time, the datastore can be in many different unreference states
+        await delay(approximateUnreferenceTimestampMs);
+
+        // Load a new container and summarizer based on the latest summary, summarize
+        const {
+            container: summarizingContainer2,
+            summarizer: summarizer2,
+        } = await loadSummarizerAndContainer(summaryVersion);
+
+        return {
+            absolutePath: blobHandle.absolutePath,
+            summarizingContainer: summarizingContainer2,
+            summarizer: summarizer2,
+            summaryVersion,
+        };
+    };
+
+    // If this test starts failing due to runtime is closed errors try first adjusting `sweepTimeoutMs` above
+    itExpects("Handle request for tombstoned blobs fails in summarizing container loaded after sweep timeout",
```

Comment:
This test is for non-summariizing container right. So, the test title needs to be updated.